appvm-scripts/lib/udev/rules.d/90-qubes-virtual-input-device.rules

```diff
@@ -0,0 +1 @@
+ATTRS{name}=="Qubes Virtual Input Device", ACTION=="add", GROUP="user"
```

Comment:
This file is not installed anywhere (add it to makefile, and then both rpm spec file and debian "install" file).
And it should be `GROUP="qubes"`